### PR TITLE
`users/id/playlists` endpoint

### PR DESCRIPTION
**Pull request by hayribakici**
This PR implements the `users/{userId}/playlists` endpoint.

### CODE REVIEW

File: test/spotify_test.dart [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import 'dart:async';
+import 'dart:math';
```
Comment (rinukkusu) [inline test/spotify_test.dart:2]: What math are we doing here?

File: test/spotify_test.dart [hunk 2]
```diff
@@ -109,7 +125,7 @@ Future main() async {
 
   group('Show episodes', () {
     test('list', () async {
-      var episodes = await spotify.shows.episodes('4AlxqGkkrqe0mfIx3Mi7Xt');
+      var episodes = spotify.shows.episodes('4AlxqGkkrqe0mfIx3Mi7Xt');
```
Comment (rinukkusu) [inline test/spotify_test.dart:128]: Why remove the `await`? 🤔

**Review by rinukkusu — COMMENTED** [review]
Looks mostly good to me!

File: test/spotify_test.dart (hunk 1 above)
Comment (hayribakici) [reply test/spotify_test.dart:2]: well well, who snuck in here.

File: test/spotify_test.dart (hunk 2 above)
Comment (hayribakici) [reply test/spotify_test.dart:128]: I removed, because vscode showed me a warning, that the `await` is applied to a `Pages<>` which is not a  `Future`.

Comment (rinukkusu) [reply test/spotify_test.dart:127]: Makes sense 😄

**Review by rinukkusu — APPROVED** [review]
LGTM

**Pull request merged**